sdk/identity/Azure.Identity/TROUBLESHOOTING.md

```diff
@@ -1,270 +1,222 @@
-﻿## Troubleshooting Azure Identity Authentication Issues
+﻿# Troubleshooting Azure Identity Authentication Issues
 
-The Azure Identity SDK offers various `TokenCredential` implementations. The most common exceptions observed for failure scenarios
-are `CredentialUnavailableException`
-and `AuthenticationFailedException`.
-
-- `CredentialUnavailableException` indicates that the credential cannot execute in the current environment setup due to lack of required configuration.
-- `AuthenticationFailedException` indicates that the credential was able to run/execute but ran into an authentication issue from the server's end. This can happen
-  due to invalid configuration/details passed in to the credential at construction time.
-
-This troubleshooting guide covers mitigation steps to resolve these exceptions thrown by various `TokenCredential` implementations in the Azure Identity .NET client
-library.
+This troubleshooting guide covers failure investigation techniques, common errors for the credential types in the Azure Identity .NET client library, and mitigation steps to resolve these errors.
 
 ## Table of contents
+- [Handling Azure Identity Exceptions](#handling-azure-identity-exceptions)
+  - [AuthenticationFailedException](#authenticationfailedexception)
+  - [CredentialUnavailableException](#credentialunavailableexception)
+- [Finding Relevant Information in Exception Messages](#finding-relevant-information-in-exception-messages)
+- [Enabling and Configuring Logging](#enabling-and-configuring-logging)
+- [Troubleshooting DefaultAzureCredential Authentication Issues](#troubleshooting-defaultazurecredential-authentication-issues)
```

Comment:
I had received suggestions previously from @scottaddie to make these links as `troubleshoot` instead of `troubleshooting`. We follow that convention for JS.

Reply:
That's correct. Here are some reasons:

1. It's harder for non-English speakers to parse since many don't have this word form (e.g., Ich gehe and Me voy is "I am going" in German and Spanish, respectively).
2. Gerunds are longer.
3. In procedural content especially, the indicative verb form is really the most direct (create a database vs. creating a database).

Reply:
@scottaddie Should we change this for all the headers and TOC entries? For instance should `Finding Relevant Information in Exception Messages` be `Find Relevant Information in Exception Messages`? What about the title of this document `Troubleshooting Azure Identity Authentication Issues` should be `Troubleshoot Azure Identity Authentication Issues`? The file name is `TROUBLESHOOTING.md`.

Reply:
The file name is fine. We should, however, change the headers and ToC entries.